ui/src/features/annotator/loading.component.tsx

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+import { Flex, Heading, View } from '@geti/ui';
+
+import IntelBrandedLoadingGif from '../../assets/intel-loading.webp';
+
+export const Loading = ({ isLoading }: { isLoading: boolean }) => {
```

Comment:
This is a pretty generic name, and from the code, I understand that this component specifically applies to sam images only.  `ImageProcessingSpinner` or `ImageProcessingLoader`

Reply:
It will be used by any tool or event that requires the annotator to be in a loading state. I'll add a "description" prop to make it more generic